src/main/java/cam72cam/mod/util/ModCoreCommand.java

```diff
@@ -1,15 +1,19 @@
 package cam72cam.mod.util;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Consumer;
+
+import org.codehaus.plexus.util.StringUtils;
```

Comment:
Would you mind switching this over to apache's string utils?  I try to minimize the number of libraries UMC directly depends upon (even if present in MC)

http://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNumeric-java.lang.CharSequence-

Reply:
Yeah, sure, no problem

Reply:
Kinda went under my radar.
I thought i used the Apache StringUtils already..